stream_alert/shared/alert_table.py

```diff
@@ -100,38 +100,40 @@ def pending_alerts(self, rule_name, alert_proc_timeout_sec):
                 This is used to determine whether an alert could still be in progress
 
         Yields:
-            Alert: An alert instance for each row in the Dynamo table
+            dict: Each row in the Dynamo table which is not being worked on by the alert processor.
         """
+        time_to_retry = datetime.utcnow() - timedelta(seconds=alert_proc_timeout_sec)
```

Comment:
as we discussed in person, it might be helpful to rename this var since I was confused by it - your call!

Reply:
Yeah, I'll rename to `in_progress_threshold` and add some more comments